HexCtrl/src/Dialogs/CHexDlgOpers.cpp

```diff
@@ -92,10 +96,30 @@ BOOL CHexDlgOpers::OnCommand(WPARAM wParam, LPARAM lParam)
 		}
 		CheckRadioButton(IDC_HEXCTRL_OPERS_RADIO_OR, IDC_HEXCTRL_OPERS_RADIO_FLOOR, iRadioID);
 	}
+	else if (wID >= IDC_HEXCTRL_OPERS_RADIO_BYTE
+			&& wID <= IDC_HEXCTRL_OPERS_RADIO_QWORD)			
+	{
+		bool bEnabled { };
+		switch (wID)
+		{
+		case IDC_HEXCTRL_OPERS_RADIO_BYTE:
+			bEnabled = false;			
+			break;
+		case IDC_HEXCTRL_OPERS_RADIO_WORD:
+		case IDC_HEXCTRL_OPERS_RADIO_DWORD:
+		case IDC_HEXCTRL_OPERS_RADIO_QWORD:
+			bEnabled = true;
+			break;
+		default:
+			fHere = false;
+		}
+		GetDlgItem(IDC_HEXCTRL_OPERS_RADIO_LE)->EnableWindow(bEnabled);
+		GetDlgItem(IDC_HEXCTRL_OPERS_RADIO_BE)->EnableWindow(bEnabled);
+	}
 	else
 		fHere = false;
 
-	return fHere ? TRUE : CDialogEx::OnCommand(wParam, lParam);
+	return fHere ? true : CDialogEx::OnCommand(wParam, lParam);

```

Comment:
What is this change for?
The function `OnCommand` returns `BOOL` not `bool`, so `TRUE` is fully justified.
The `true` would mean implicit conversion to `int`.

Reply:
That change was developer "auto-pilot". You are correct. I apologise.